### PR TITLE
Support for kernel version >= 5.0.0

### PR DESCRIPTION
**Pull request by Nagefire**
I updated a few sections of the code to make it compatible with the newest release of the linux kernel, tested on arch linux with kernel 5.0.0

### CODE REVIEW

**Comment by kaduke** [comment]
LGTM

File: PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+pkgdesc="A driver for Ralink Mediatek mt766u, mt7632u, and mt7612u chipsets"
+arch=('any')
+url="https://github.com/kaduke/NetGear-A6210"
+license=('GPL')
+
+depends=('linux-headers')
+
+optdepends=('dkms: Automatically rebuild drivers upon updating the kernel')
+
+source=("${pkgname}-${pkgver}.tar.gz::https://gitlab.com/Nagefire/${pkgname}/raw/master/tar/${pkgname}-${pkgver}.tar.gz")
```
Comment (kaduke) [inline PKGBUILD]: Sorry for my delay in reviewing this. I appreciate the contribution! 

This source doesn't seem to exist anymore. Do you still want to include a `PKGBUILD`?

**Review by kaduke — CHANGES_REQUESTED** [review]

File: PKGBUILD (hunk 1 above)
Comment (Nagefire) [reply PKGBUILD]: No, you can remove it

**Review by kaduke — APPROVED** [review]

**Comment by kaduke** [comment]
Thanks @Nagefire!

**Pull request merged**